### ISSUE TITLE
Prevent auto-import for Prim members

### ISSUE THREAD

**Issue by zaboco** [intellij-purescript/intellij-purescript#387]
When I want to type a builtin type in the IDE (e.g. `Array` or `Boolean`), I get a completion suggestion, which is fine. 

<img width="204" alt="image" src="https://github.com/intellij-purescript/intellij-purescript/assets/983819/2358465e-6e37-45b8-9b07-efcd1eba6f71">

The issue is that once I accept the completion, that type is also imported, even if it doesn't need to be, since `Prim` is already available:

```purescript
import Prim (class Boolean)
```

And even worse, I get a compile error:

```
Cannot import type class Boolean from module Prim
  It either does not exist or the module does not export it.
```

---

Versions:
- IntelliJ IDEA 2023.2.3
- Purescript plugin 0.80.17

**Comment by Neppord** [comment]
I see this as two issues:
1. The plugin imports things already in scope, if not the exact import already exists.
2. It imports Boolean as a class, but should be a data constructor (?)

**Comment by zaboco** [comment]
Yes, for 1. The plugin is not aware that Prim is already imported by the compiler.
For 2. it should be `import Prim (Boolean)`, indeed. But this mismatch (importing `class Boolean` instead) only happens in this case. E.g. if I define `data Bool = ...` in one of my modules, it correctly imports it in other modules.

**Comment by Neppord** [comment]
fixed in [v0.80.19](https://plugins.jetbrains.com/plugin/9738-purescript/edit/versions/stable/440024)
